src/managers/TileManager.ts

```diff
@@ -47,6 +49,28 @@ class TileManager {
         );
     }
 
+    public getStartingTiles(): Tile[] {
+        const session: Nullable<SessionDTO> = SessionManager.get();
+        assert(session);
+        const tiles: Tile[] = [];
+        const random: seedrandom.PRNG = seedrandom(session.seed);
+        const veins: int[] = seededShuffle([9, 10, 10], session.seed); // 9 is gold, 10 are coal veins
+
+        for (const item of preplacedTiles) {
+            const tile: Tile = new Tile(random, item.type || veins.pop()!);
+            const tileDTO: TileDTO = {
+                id: tile.id,
+                rotation: item.rotation,
+                coordinateX: item.coordinateX,
+                coordinateY: item.coordinateY,
+            };
```

Comment:
Good practice: When manually creating a object with { }... add the desired type:
```
    ...
} as TileDTO;
```

Reply:
Noted. But why? Is the type definition itself not enough?

Reply:
its more like water proofing... the type for the variable makes sure that the variable will always have this type and the "as ..." by the object ensures that the value its self will always have the type and the inner properties get checked by the type properties